### PR TITLE
Decanter 161 page layout  3 column

### PR DESCRIPTION
**Pull request by bixgomez**
# READY FOR REVIEW
- This PR satisfies the basic four layout profiles: single column, left sidebar, right sidebar, and three column.
- Markup for page layouts might change in the future, depending on overall site layouts. Could be simplified a bit.
- Please note that Code Canyon is complaining about an svg/png fallback that predates this work.

# Summary
- Implementing responsive grids: single column, two column (left & right sidebar), and three column.

# Steps to Test
1. Install and compile Style Guide (though compiling shouldn't be necessary)
1. Visit the "templates" area of Decanter: [Decanter Templates](http://decanter.stanford.loc/section-templates.html)
1. Analyze grids at various breakpoints for visual accuracy
1. Compare against UComm design guidelines: [Design Guidelines](https://www.figma.com/file/GytUeUKUtlYFdSkOBdVhJUKy/Interior-Pages)

# Jira Tickets
- [Page Layout: Single Column](https://stanfordits.atlassian.net/browse/UCP-483)
- [Page Layout: Left Sidebar](https://stanfordits.atlassian.net/browse/UCP-484)
- [Page Layout: Right Sidebar](https://stanfordits.atlassian.net/browse/UCP-485)
- [Page Layout: Three Column](https://stanfordits.atlassian.net/browse/UCP-486)
- [Add new breakpoints](https://stanfordits.atlassian.net/browse/UCP-482)

# GitHub Issues
- [Upgrade Neat to 3.0](https://github.com/SU-SWS/decanter/issues/159)
- [Page Layout: Single Column](https://github.com/SU-SWS/decanter/issues/164)
- [Page Layout: Left Sidebar](https://github.com/SU-SWS/decanter/issues/163)
- [Page Layout: Right Sidebar](https://github.com/SU-SWS/decanter/issues/162)
- [Page Layout: Three Column](https://github.com/SU-SWS/decanter/issues/161)
- [Add new breakpoints](https://github.com/SU-SWS/decanter/issues/168)

# Other Associated Links
- [Decanter Templates](http://decanter.stanford.loc/section-templates.html)
- [Design Guidelines](https://www.figma.com/file/GytUeUKUtlYFdSkOBdVhJUKy/Interior-Pages)
- [Responsive Notes](https://www.figma.com/file/gCGDmlyhKJ88zLvVcoUBYjjK/UComm-Design-Library?node-id=279%3A0)

### CODE REVIEW

**Comment by JBCSU** [comment]
@bixgomez When I look at the **Page Layout** templates embedded in the style guide, they look ok. But when I click on the button to open each layout in a separate window, which I think is a more realistic test of the page layouts, the text goes edge-to-edge in the browser window, and the grid background extends off both the right and left sides of the window, resulting in a horizontal scroll bar.

**Comment by bixgomez** [comment]
In response to Joe's comments:

Yes, yikes, as both you and JB noted, I see that scrollbar.  A consequence of using grid-collapse without an overflow-h: hidden (I think).  I am collapsing the grid in the interest of allowing the overall SITE layout to have its own gutters -- I am not a fan of inner grids having any outer padding, so I'm trying to avoid that....

Regarding Decanter grid classes -- yes, I can definitely use them if that is preferable.  I had thought that those classes were only for special cases (like for use in views markup etc.) -- sorry I did not realize they were there for us to use in our own work as well!

That extra dash is intentional -- I wanted the "global" Sass file to appear on top, in the interest of convenience while coding.

**Comment by JBCSU** [comment]
@bixgomez re 

> I am collapsing the grid in the interest of allowing the overall SITE layout to have its own gutters

The associated issues are for creating page layouts, which, as I understand it, would include site-wide gutters. I think creating re-usable grids that can be embedded, as you have done, is a fine approach. We probably just need a wee bit of additional styling, e.g. `overflow-h: hidden` to say "Ok, this grid is the grid for the whole page."

**Comment by JBCSU** [comment]
Regarding the discussion over Decanter classes: I really, really do not want to use magic classes in my theme, so I'm really, really hoping that all functionality is coded in some combination of variables, mixins and functions, and that the decanter classes are all just wrappers around mixins, etc. I get that I might have to use a magic class here and there, but I really hope that our goal is to honor the spirit of Bourbon and Neat, and supply all functionality through mixins.

**Comment by josephgknox** [comment]
+1 JB. I think that is - and should be - our goal. And, if that means keeping Decanter grid classes out of the page layout markup, then let's not put them in there. Currently, the Decanter grid classes are wrapped around mixins. They can be available to serve those who want to create layouts through the UI, but won't ever be used to hardcode a layout in a template.

That said, are we good, then, with using the column, etc. classes attached to mixins that are in the template files for these layouts now (as part of this PR)? We'll use the necessary Neat mixins inside our components/patterns to determine their layout?

**Comment by josephgknox** [comment]
Re: the site-wide gutters discussion. I am also OK with this approach. So, to confirm, Decanter will never has default gutters for a site set, rather those will be set in our own themes and/or individual projects, yes?

**Comment by JBCSU** [comment]
@josephgknox 

> So, to confirm, Decanter will never has default gutters for a site set, rather those will be set in our own themes and/or individual projects, yes?

Actually, that's the opposite of what I intended. I think Decanter should have default gutters as specified in our shared design docs, but that the values should, of course, be override-able in our themes.

**Comment by josephgknox** [comment]
@JBCSU 

That's a relief. I am much more of a fan of that. And, yes, ensuring that Decanter can be overridden in partials in our themes/on projects will always be important. Thanks for the confirmation and clarification.

Do we want default outter gutters defined, too? Sounds like there is an argument against that.

File: scss/components/templates/_page-layout---global.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.page-layout {
+  max-width: $site-max-width;
+  margin: 0 auto;
+
+  @include grid-media($grid-xs) {
+    @include grid-container($grid-xs);
```
Comment (sherakama) [inline scss/components/templates/_page-layout---global.scss]: I don't think we need to set the class as a container for each breakpoint. I believe you can move this outside of the grid-media wrapper.

File: scss/components/templates/_page-layout---global.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @include grid-visual;
+  }
+
+  @include grid-media($grid-md) {
+    @include grid-container($grid-md);
+    @include grid-visual;
```
Comment (sherakama) [inline scss/components/templates/_page-layout---global.scss]: the visual mixin is fine for preview but we will want to remove this before merging.

File: scss/components/templates/_page-layout--single-column.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// Page Layout: Single column
+//
+// A basic, responsive single column page layout.
+//
+// Markup: ../../templates/components/templates/page-layout--single-column/page-layout--single-column.twig
+//
+// Style guide: Templates.Page Layout: Single column
+//
+
+.page-layout--single-column {
```
Comment (sherakama) [inline scss/components/templates/_page-layout--single-column.scss]: Shouldn't we declare the topmost wrapper as a container?

File: scss/decanter-grid.scss [hunk 4]
```diff
@@ -26,7 +26,8 @@ $border-box-sizing: false;
 // 2. Vendors.
 @import
   'bourbon',
-  'neat';
+  'neat',
+  'breakpoint';
```
Comment (sherakama) [inline scss/decanter-grid.scss]: Please remove breakpoint.

File: scss/decanter.scss [hunk 5]
```diff
@@ -21,7 +21,8 @@
 @import
   'bourbon',
   'neat',
-  'normalize';
+  'normalize',
+  'breakpoint';
```
Comment (sherakama) [inline scss/decanter.scss]: Please remove breakpoint.

File: scss/elements/_typography.scss [hunk 6]
```diff
@@ -136,7 +136,7 @@ dfn {
   }
 
   > :first-child {
-    @include media($medium-screen) {
+    @media #{$breakpoint-md} {
```
Comment (sherakama) [inline scss/elements/_typography.scss]: I'd rather see this use neat's grid for consistency rather than a raw mediaquery.

File: scss/utilities/kss/_kss.scss [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+//
+// KSS Fullscreen Scrolling Fix
+//
+// Adding 2px to the default kss fullscreen padding. This resolves the 1-2px horizontal scrolling issue.
+//
+// Style guide: KSS.ScrollFix
+//
+
+#kss-node .kss-section.is-fullscreen {
```
Comment (sherakama) [inline scss/utilities/kss/_kss.scss]: This is specific to the styleguide and does not belong here. Please remove.

File: scss/utilities/kss/index.scss [hunk 8]
```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
```
Comment (sherakama) [inline scss/utilities/kss/index.scss]: This is specific to the styleguide and does not belong here. Please remove.

File: scss/utilities/mixins/grid/_allow-layout-classes.scss [hunk 9]
```diff
@@ -11,45 +11,45 @@
   @include margin(null auto);
 
   &.width-one-half {
-    @include media($medium-screen) {
+    @media #{$breakpoint-md} {
       width: 50%;
```
Comment (sherakama) [inline scss/utilities/mixins/grid/_allow-layout-classes.scss]: Should we refactor these to use the grid system instead of percentages?

File: scss/variables/components/_alerts.scss [hunk 10]
```diff
@@ -15,7 +15,9 @@
 //
 // Styleguide: Variables.Components.Alerts.decanter-custom-alerts
 //
-$decanter-custom-alerts: () !default;
+$decanter-custom-alerts: (
+    sample: rgba(125, 125, 125, 0.5), // sample default value
```
Comment (sherakama) [inline scss/variables/components/_alerts.scss]: Why is there a default value if it is only a sample? This would be better as a comment if not intended for use.

File: scss/variables/core/_breakpoints.scss [hunk 11]
```diff
@@ -7,94 +7,196 @@
...
+// $screen-xs
 //
-// Markup: $small-screen:  481px !default;
+// Markup: $screen-xs: 0px !default;
 //
-// Styleguide: Variables.Core.Breakpoints.small-screen
+// Styleguide: Variables.Core.Breakpoints.screen-xs
 //
-$small-screen: 481px !default;
+$screen-xs: 0 !default;
```
Comment (sherakama) [inline scss/variables/core/_breakpoints.scss]: Shouldn't this be the smallest breakpoint greater than 0?

File: scss/variables/core/_breakpoints.scss [hunk 12]
```diff
@@ -7,94 +7,196 @@
...
+// Styleguide: Variables.Core.Breakpoints.xl
+//
+// $breakpoint-xl: (media: 'only screen and (min-width: #{$screen-xl})') !default;
+$breakpoint-xl: 'only screen and (min-width: #{$screen-xl})' !default;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 //
-// $small
+// $grid-xs
```
Comment (sherakama) [inline scss/variables/core/_breakpoints.scss]: Where is grid-sm and grid-xl?

File: scss/variables/core/_magic-numbers.scss [hunk 13]
```diff
@@ -118,6 +118,17 @@ $focus-shadow: 0 0 3px $color-cool-grey, 0 0 7px $color-cool-grey !default;
 //
 $nav-width: 951px !default;
 
+//
+// $breakpoint-navwidth
```
Comment (sherakama) [inline scss/variables/core/_magic-numbers.scss]: This seems related to just the nav.. @josephgknox I cannot remember what we decided. All variables in the core file or can we have specifics in each mixin?

File: templates/components/molecules/icon-grid/icon-grid.twig [hunk 14]
```diff
@@ -6,7 +6,7 @@
   {% if items is not empty %}
   {% set classes = "" %}
     {% for item in items %}
-      {% include "@decanter/components/molecules/media-block-row/media-block-row.twig" with item %}
+      {% include "../../molecules/media-block-row/media-block-row.twig" with item %}
```
Comment (sherakama) [inline templates/components/molecules/icon-grid/icon-grid.twig]: Where did the namespace go? This is important to keep if we plan to use the templates in other systems.

File: templates/components/molecules/media-block-row/media-block-row.twig [hunk 15]
```diff
@@ -6,7 +6,7 @@
 
   {% if items not empty %}
     {% for item in items %}
-      {% include "@decanter/components/molecules/media-block/media-block.twig" with item %}
+      {% include "../../molecules/media-block/media-block.twig" with item %}
```
Comment (sherakama) [inline templates/components/molecules/media-block-row/media-block-row.twig]: Where did the namespace go? This is important to keep if we plan to use the templates in other systems.

File: templates/components/molecules/media-block/media-block.twig [hunk 16]
```diff
@@ -1,4 +1,4 @@
 <div{{ attributes }} class="decanter-media_block {{ classes }}">
-  {% include "@decanter/components/atoms/media-block-img/media-block-img.twig" with { "classes": '' } %}
-  {% include "@decanter/components/atoms/media-block-body/media-block-body.twig" with { "classes": '' } %}
+  {% include "../../atoms/media-block-img/media-block-img.twig" with { "classes": '' } %}
```
Comment (sherakama) [inline templates/components/molecules/media-block/media-block.twig]: Where did the namespace go? This is important to keep if we plan to use the templates in other systems.

File: templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig [hunk 17]
```diff
@@ -0,0 +1,11 @@
+<div class="page-layout page-layout--left-sidebar">
```
Comment (sherakama) [inline templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig]: Probably should add a variable to each of the div containers so that other's may add properties to the divs. 
Use case: As another developer I want to add the purple class to the outer wrapper and the wide class to the content div.

File: templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig [hunk 18]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="column column--head">
+    {{ headContent }}
```
Comment (sherakama) [inline templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig]: Twig variable naming conventions use "_" separators and not camel case.

File: templates/components/templates/page-layout--right-sidebar/page-layout--right-sidebar.twig [hunk 19]
```diff
@@ -0,0 +1,11 @@
+<div class="page-layout page-layout--right-sidebar">
+  <div class="column column--head">
+    {{ headContent }}
```
Comment (sherakama) [inline templates/components/templates/page-layout--right-sidebar/page-layout--right-sidebar.twig]: Twig variable naming conventions use "_" separators and not camel case.

File: templates/components/templates/page-layout--three-column/page-layout--three-column.twig [hunk 20]
```diff
@@ -0,0 +1,14 @@
+<div class="page-layout page-layout--three-column">
+  <div class="column column--head">
+    {{ headContent }}
```
Comment (sherakama) [inline templates/components/templates/page-layout--three-column/page-layout--three-column.twig]: Twig variable naming conventions use "_" separators and not camel case.

File: scss/components/molecules/_navigation.scss [hunk 21]
```diff
@@ -23,7 +23,7 @@
   width: $sliding-panel-width;
   z-index: $z-index-nav;
 
-  @include media($nav-width) {
+  @media #{$breakpoint-navwidth} {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/components/molecules/_navigation.scss]: Ultimately, yes -- but this is an existing media query, so I needed to do this in order to avoid compilation errors.

File: scss/components/molecules/_navigation.scss [hunk 22]
```diff
@@ -68,15 +68,15 @@
   @include font-smoothing;
 
   &:hover {
-    @include media($nav-width) {
+    @media #{$breakpoint-navwidth} {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/components/molecules/_navigation.scss]: Ultimately, yes -- but this is an existing media query, so I needed to do this in order to avoid compilation errors.

File: scss/components/molecules/_navigation.scss [hunk 23]
```diff
@@ -68,15 +68,15 @@
...
       @include nav-border;
     }
   }
 }
 
 .decanter-nav__container {
-  @include media($nav-width) {
-    @include outer-container;
+  @media #{$breakpoint-navwidth} {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/components/molecules/_navigation.scss]: Ultimately, yes -- but this is an existing media query, so I needed to do this in order to avoid compilation errors.

File: scss/components/molecules/_navigation.scss [hunk 24]
```diff
@@ -89,13 +89,13 @@
 
   order: 2;
 
-  @include media($nav-width) {
+  @media #{$breakpoint-navwidth} {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/components/molecules/_navigation.scss]: Ultimately, yes -- but this is an existing media query, so I needed to do this in order to avoid compilation errors.

File: scss/components/templates/_page-layout---global.scss (hunk 2 above)
Comment (bixgomez) [reply scss/components/templates/_page-layout---global.scss]: OK

File: scss/components/templates/_page-layout--left-sidebar.scss [hunk 25]
```diff
@@ -0,0 +1,61 @@
...
+// Page Layout: Left sidebar
+//
+// A basic, responsive 2-column (with left sidebar) page layout.
+//
+// Markup: ../../templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig
+//
+// Style guide: Templates.Page Layout: Left sidebar
+//
+
+.page-layout--left-sidebar {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/components/templates/_page-layout--left-sidebar.scss]: I left them out because they are identical.  I suppose, if they ever change, they'd be really and truly missing, so I can stick them back in.

File: package.json [hunk 26]
```diff
@@ -34,7 +34,8 @@
     "grunt-symlink": "^0.4.0",
     "grunt-verbosity": "^1.0.1",
     "jquery": "^3.3.1",
-    "kss": "~3.0.0-beta.13",
-    "postcss-cli": "^5.0.0"
+    "kss": "~3.0.0-beta.22",
+    "postcss-cli": "^5.0.0",
+    "breakpoint-sass": "^2.7.1"
```
[the remark this answers is not in the document]
Comment (JBCSU) [reply package.json]: By 'this' you mean `breakpoint-sass`, right?

File: scss/core/grid/_offsets.scss [hunk 27]
```diff
@@ -10,50 +10,51 @@
 // Required if grid-box contains multiple rows.
 // Required if browser does not support :last-child.
 .decanter-end-row {
-  @include omega;
+  // RJG: More research is needed!
+  // @include omega;
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/core/grid/_offsets.scss]: I am not sure!  I think it might be -- I haven't quite figured out what it did, but having it in causes errors now.

File: scss/decanter-grid.scss (hunk 4 above)
Comment (bixgomez) [reply scss/decanter-grid.scss]: OK, yes, will do!

File: package.json (hunk 26 above)
[the remark this answers is not in the document]
Comment (josephgknox) [reply package.json]: I can confirm that is what he mean by "this." Because we're no longer using it, there should be no issue with removing it from the package. Richard and I discussed this earlier today.

File: scss/decanter.scss (hunk 5 above)
Comment (bixgomez) [reply scss/decanter.scss]: OK, yes, will do!

File: scss/variables/components/_alerts.scss (hunk 10 above)
Comment (bixgomez) [reply scss/variables/components/_alerts.scss]: With Neat 3.0, we need at least one value in there, or we get an error.

File: scss/variables/core/_breakpoints.scss (hunk 11 above)
Comment (bixgomez) [reply scss/variables/core/_breakpoints.scss]: That's how I always do it, where 0 is baseline and xs is something like 320px -- so, I can easily change this.  I believe I did this based on how Figma was laid out.

File: templates/components/molecules/media-block/media-block.twig (hunk 16 above)
Comment (bixgomez) [reply templates/components/molecules/media-block/media-block.twig]: @sherakama : I was looking for that namespace!  I don't know where it went either, perhaps we can look into that together.

File: templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig (hunk 17 above)
Comment (bixgomez) [reply templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig]: WILL WORK ON THIS.

File: Gruntfile.js [hunk 28]
```diff
@@ -51,7 +51,10 @@ module.exports = function(grunt) {
     watch: {
       css: {
         files: '**/*.scss',
-        tasks: ['sass', 'postcss', 'styleguide']
+        tasks: ['sass', 'postcss', 'styleguide'],
+        options: {
+          livereload: true
```
Comment (sherakama) [inline Gruntfile.js]: Neat, thanks.

File: css/decanter-grid.css [hunk 29]
```diff
@@ -1,3 +1,6 @@
+#kss-node .kss-section.is-fullscreen {
```
Comment (sherakama) [inline css/decanter-grid.css]: This markup is for the styleguide only and should not be rendered to the grid library.

File: templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig (hunk 18 above)
Comment (bixgomez) [reply templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig]: Thanks for the heads_up!  Fixed.

File: templates/components/templates/page-layout--right-sidebar/page-layout--right-sidebar.twig (hunk 19 above)
Comment (bixgomez) [reply templates/components/templates/page-layout--right-sidebar/page-layout--right-sidebar.twig]: Thanks for the heads_up!  Fixed.

File: templates/components/templates/page-layout--three-column/page-layout--three-column.twig (hunk 20 above)
Comment (bixgomez) [reply templates/components/templates/page-layout--three-column/page-layout--three-column.twig]: Thanks for the heads_up!  Fixed.

File: css/decanter.css [hunk 30]
```diff
@@ -1,9 +1,3 @@
-html {
```
Comment (sherakama) [inline css/decanter.css]: Is this no longer a thing? Something to do with normalize?

File: css/decanter.css [hunk 31]
```diff
@@ -1989,4 +1910,465 @@ a.decanter-button--unstyled {
   border-bottom: 1px solid #b6b1a9;
   border-left: 1px solid #b6b1a9;
   border-right: 1px solid #b6b1a9; }
+
+.page-layout {
```
Comment (sherakama) [inline css/decanter.css]: This leads to a good question. How much of this page layout code do we want in the default compiled css? Would we rather have it optional?

File: css/decanter-no-markup.css [hunk 32]
```diff
@@ -1,2 +1,4 @@
+#kss-node .kss-section.is-fullscreen {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply css/decanter-no-markup.css]: I'll work on making that more specific.

File: scss/core/grid/_base.scss [hunk 33]
```diff
@@ -7,15 +7,15 @@
...
+  @include grid-container;
 }
 
 .decanter-grid {
   @include padding(null $site-margins-mobile);
 
   max-width: $site-max-width;
 
-  @include media($medium-screen) {
+  @media #{$breakpoint-md} {
```
[the remark this answers is not in the document]
Comment (bixgomez) [reply scss/core/grid/_base.scss]: this is an existing simple media query, so I needed to do this in order to avoid compilation errors.

File: scss/elements/_typography.scss (hunk 6 above)
Comment (bixgomez) [reply scss/elements/_typography.scss]: this is an existing simple media query, so I needed to do this in order to avoid compilation errors.

File: scss/utilities/kss/_kss.scss (hunk 7 above)
Comment (bixgomez) [reply scss/utilities/kss/_kss.scss]: I'll work on that shortly.

File: css/decanter.css (hunk 30 above)
Comment (josephgknox) [reply css/decanter.css]: Yep, this is from Normalize, which we're still using. That said, I'd say that it's still a thing.

https://necolas.github.io/normalize.css/8.0.0/normalize.css

File: scss/variables/core/_breakpoints.scss (hunk 11 above)
Comment (sherakama) [reply scss/variables/core/_breakpoints.scss]: Ah, I just looked at the figma docs again. xs is set to 0. So there really is only 4 breakpoints. This is ok to keep it at 0. I suspect the `mq` will be for the range between 0 and grid-sm.

File: templates/components/molecules/media-block/media-block.twig (hunk 16 above)
Comment (sherakama) [reply templates/components/molecules/media-block/media-block.twig]: Yeha, this is something I can help with. It probably has something to do with either this not getting called: https://github.com/SU-SWS/decanter/blob/master/kss-config.json or a syntax difference in versions.

Comment (JBCSU) [reply templates/components/molecules/media-block/media-block.twig]: I noticed that this syntax stopped working after upgrading kss. I didn't see any doc in the change log explaining it, tho.

File: css/decanter-grid.css (hunk 29 above)
Comment (josephgknox) [reply css/decanter-grid.css]: +1

File: css/decanter-no-markup.css (hunk 32 above)
[the remark this answers is not in the document]
Comment (josephgknox) [reply css/decanter-no-markup.css]: Is this added to control layout in the style guide? If so, we can create a seperate stylesheet and/or apply this elsewhere. This shouldn't live in the Decanter code output that others will see and use on their projects.

File: css/decanter.css (hunk 31 above)
Comment (josephgknox) [reply css/decanter.css]: If the code makes up the page layouts that we've agreed on, I would lean towards it living in the default compiled CSS. Of course, we'll want to be sure that the layouts can be overridden like other things in partials. Were you thinking we might not want to do this? Happy to hear the other side of the argument.

**Comment by sherakama** [comment]
Closing and pivoting to use grid/flexbox.
